### PR TITLE
DOC: Content Library dropdown selector

### PR DESCRIPTION
**Pull request by catong**
This PR contains doc changes for SOL-123. The requirement to manually enter a library ID or IDs in a randomized content block was changed so that users now use a dropdown to select only a single library name. @bradenmacdonald @antoviaque @marcotuts please review for technical accuracy. Thanks!
SOL-457

### CODE REVIEW

**Comment by bradenmacdonald** [comment]
:+1:

**Comment by catong** [comment]
@mhoeber @lamagnifica @srpearce Can one of you please do an editorial review? Thanks!

File: en_us/shared/building_and_running_chapters/creating_content/libraries.rst [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+You see the new library, to which you can now add components. For details about
+adding components to a library, see :ref:`Add Components to a Library`.
+
+The system-generated Library ID for the new library appears in the sidebar.
+
+  .. image:: ../../../shared/building_and_running_chapters/Images/ContentLibraryID.png
+     :alt: The Library ID for the new library is shown the sidebar
+
+.. note:: Copy or make a note of the Library ID for the new library. When you
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/creating_content/libraries.rst:112]: I need to remove this note; no longer required with drop-down selector.

**Comment by srpearce** [comment]
:+1:

**Comment by catong** [comment]
Doc task is complete. Waiting to merge doc when the feature is live on edx.org.

**Comment by catong** [comment]
Closing this PR: combined in https://github.com/edx/edx-documentation/pull/207

**Pull request closed without merging**